opencti-platform/opencti-graphql/tests/08-httpTaxii/httpTaxii-test.ts

```diff
@@ -0,0 +1,154 @@
+    }).rejects.toThrowError('Request failed with status code 401');
+
+    /*
+    UNCOMMENT THIS CODE when fixing taxii2 push server behavior on no content type or bad content type
+        const headersToTest1 = {
+      Authorization: `Bearer ${ADMIN_API_TOKEN}`
+    };
+    await expect(async () => {
+      await axios.post(`${getBaseUrl()}/taxii2/root/collections/${taxiiPushIngestion.id}/objects`, bundleObject, { headers: headersToTest1 });
+    }).rejects.toThrowError('????');
+    */
+    const headersToTest2 = {
+      'Content-Type': 'application/taxii+json;version=2.1',
+      Authorization: `Bearer ${ADMIN_API_TOKEN}`
+    };
+
+    let axiosResponsePost = await axios.post(`${getBaseUrl()}/taxii2/root/collections/${taxiiPushIngestionId}/objects`, bundleObject, { headers: headersToTest2 });
+    expect(axiosResponsePost.status, 'With correct content type and authentication should works fine').toBe(200);
+    // We do not check entity from bundleObject in database since it requires the worker to process it.
+
+    const headersToTest3 = {
+      'Content-Type': 'application/taxii+json; version=2.1',
+      Authorization: `Bearer ${ADMIN_API_TOKEN}`
+    };
+    axiosResponsePost = await axios.post(`${getBaseUrl()}/taxii2/root/collections/${taxiiPushIngestionId}/objects`, bundleObject, { headers: headersToTest3 });
+    expect(axiosResponsePost.status, 'With correct content type including space and authentication should works fine').toBe(200);
+
+    const headersToTest4 = {
+      'Content-Type': 'application/vnd.oasis.stix+json; version=2.1',
+      Authorization: `Bearer ${ADMIN_API_TOKEN}`
+    };
+    axiosResponsePost = await axios.post(`${getBaseUrl()}/taxii2/root/collections/${taxiiPushIngestionId}/objects`, bundleObject, { headers: headersToTest4 });
+    expect(axiosResponsePost.status, 'With an accepted content type including space and authentication should works fine').toBe(200);
+  });
+
+  it('should delete taxii post Feed', async () => {
+    await ingestionDelete(testContext, ADMIN_USER, taxiiPushIngestionId);
+  });
+});
+
+describe('Should taxii collection be exposed', () => {
+  let taxiiCollectionId: string;
+
+  it('creates a taxii collection configuration and start', async () => {
+    const dataSharingTaxii: TaxiiCollectionAddInput = {
+      name: 'Testing coverage on Taxii collection',
+      description: '',
+      authorized_members: [],
+      taxii_public: false,
+      include_inferences: true,
+      score_to_confidence: false,
+      filters: '{\'mode\':\'and\',\'filters\':[{\'key\':[\'entity_type\'],\'operator\':\'eq\',\'values\':[\'Indicator\'],\'mode\':\'or\'}],\'filterGroups\':[]}'
+    };
+    const taxiiCollection = await createTaxiiCollection(testContext, ADMIN_USER, dataSharingTaxii);
+    expect(taxiiCollection.id).toBeDefined();
+    taxiiCollectionId = taxiiCollection.id;
+    expect(taxiiCollection.name).toBe('Testing coverage on Taxii collection');
+
+    await waitInSec(10);
```

Comment:
we should poll the collection availability rather introducing a long delay that increase test duration